lnwallet/channel.go

```diff
@@ -9243,6 +9243,35 @@ func (lc *LightningChannel) RemoteBalanceDust() bool {
 	return remoteBalance <= chanState.RemoteChanCfg.DustLimit
 }
 
+// CommitBalances returns the local and remote balances in the current
+// commitment state.
+func (lc *LightningChannel) CommitBalances() (btcutil.Amount, btcutil.Amount) {
+	lc.RLock()
+	defer lc.RUnlock()
+
+	chanState := lc.channelState
+	localCommit := lc.channelState.LocalCommitment
+
+	localBalance := localCommit.LocalBalance.ToSatoshis()
+	remoteBalance := localCommit.RemoteBalance.ToSatoshis()
+
+	if chanState.ChanType.HasAnchors() && chanState.IsInitiator {
+		localBalance += 2 * AnchorSize
+	} else {
+		remoteBalance += 2 * AnchorSize
```

Comment:
I think this `else` case would be incorrect for non-anchor channels? Sure, this method is currently only used in the context of custom taproot channels, which always have anchors. But just to avoid weird future bugs.